### PR TITLE
REDCORE-446_fixing_specific_media_issues

### PR DESCRIPTION
**Pull request by Kixo**

### CODE REVIEW

File: build/media/libraries/redcore/media/redcore/js/jquery.searchtools.js [hunk 1]
```diff
@@ -1,418 +0,0 @@
...
-	// preventing against multiple instantiations
-	$.fn[pluginName] = function (options) {
-		return this.each(function () {
-			if (!$.data(this, "plugin_" + pluginName)) {
-				$.data(this, "plugin_" + pluginName, new Plugin(this, options));
-			}
-		});
-	};
-
-})(jQuery, window, document);
```
Comment (javigomez) [inline build/media/libraries/redcore/media/redcore/js/jquery.searchtools.js]: Even this file is not anymore in new installation, if you update redCORE this file remains:

screen shot 2015-11-30 at 10 43 44

We would need a php script to be executed on update to remove it.

**Comment by Kixo** [comment]
ok updated and put update script to remove old media structure files/folders...

**Comment by javigomez** [comment]
works

File: build/media/libraries/redcore/media/redcore/lib/jquery-ui/i18n/jquery-ui-i18n.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-/*! jQuery UI - v1.10.4 - 2014-01-17
+/*! jQuery UI - v1.10.3 - 2013-05-03
```
Comment (jatitoam) [inline build/media/libraries/redcore/media/redcore/lib/jquery-ui/i18n/jquery-ui-i18n.js]: Older version?

Comment (Kixo) [inline build/media/libraries/redcore/media/redcore/lib/jquery-ui/i18n/jquery-ui-i18n.js]: yes, but only for that one, newer break our code :)

**Pull request merged**